client/flutter/lib/main.dart

```diff
@@ -30,13 +30,14 @@ class _MyAppState extends State<MyApp> {
         S.delegate
       ],
       supportedLocales: S.delegate.supportedLocales,
-      theme: ThemeData(
+      theme: ThemeData.light().copyWith(
```

Comment:
I believe theme defaults to light and this is unnecessary

Reply:
It does, just being explicit. I'll revert this line.